docs/integration/cassandra.md

```diff
@@ -4,48 +4,48 @@
-bloom false positives       |  cassandra.bloom.falsepositives                     |  counter      |  long          |                                     |  The number of Bloom filter false positives
-bloom false positive ratio  |  cassandra.bloom.falsepositives.ratio               |  gauge        |  double        |                                     |  The ratio of Bloom filter false positives to total checks
-bloom disk space            |  cassandra.bloom.disk.space                         |  gauge        |  long          |  bytes                              |  Disk space used by the Bloom filters
-sstables per read           |  cassandra.number.of.sstables.per.read              |  gauge        |  double        |                                     |  Number of SSTables are accessed during a read. Includes sstables that undergo bloom-filter checks, even if no data is read from the sstable
-local write request         |  cassandra.local.write.requests                     |  counter      |  long          |                                     |  Local writes update the table's memtable and appends to a commitlog
-local write latency total   |  cassandra.local.write.requests.latency.time        |  counter      |  long          |  ms                                 |
-avg local write latency     |  cassandra.local.write.latency.avg                  |  gauge        |  double        |  ms                                 |
-local read request          |  cassandra.local.read.requests                      |  counter      |  long          |                                     |  Local reads retrieve data from a table's memtable and any necessary SSTables on disk
-local read latency total    |  cassandra.local.read.requests.latency.time         |  counter      |  long          |  ms                                 |
-avg local read latency      |  cassandra.local.read.latency.avg                   |  gauge        |  double        |  ms                                 |
+Metric Name<br> Key *(Type)* *(Unit)*                                                                     |  Description
+----------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+leaving nodes<br>**cassandra.cluster.nodes.leaving** <br>*(long gauge)*                                   |
+moving nodes<br>**cassandra.cluster.nodes.moving** <br>*(long gauge)*                                     |
+joining nodes<br>**cassandra.cluster.nodes.joining** <br>*(long gauge)*                                   |
+live nodes<br>**cassandra.cluster.nodes.live** <br>*(long gauge)* *(from the perspective of this node)*   |  Count of nodes which are visible and live
+null<br>**cassandra.cluster.nodes.unreachable** <br>*(long gauge)*                                        |
```

Comment:
"null"? I see there are few more "null"s in other places.

Reply:
fixed in GenerateDocs code. looks like missed replacing updated metrics. Will regenerate docs